SetupDataPkg/ConfDataSettingProvider/ConfDataSettingProvider.c

```diff
@@ -594,6 +594,12 @@ SettingsProviderSupportProtocolNotify (
     goto Done;
   }
 
+  if (IsSystemInManufacturingMode ()) {
+    // As promised, configuration variables will not be protected under MFG mode. Thus branch to the wrap up logic.
+    DEBUG ((DEBUG_WARN, "%a System in manufacturing mode, not applying variable policies!\n", __FUNCTION__));
+    goto Done;
+  }
+
   for (Index = 0; Index < VarListCount; Index++) {
     // Using default blob to initialize individual setting providers
     Status = RegisterSingleConfigVariable (&VarList[Index]);

```

Comment:
I think this check needs to go into RegisterSingleConfigVariable? If we dump out before it, the variable will not be set (which the profile is not enforced/set in MFG mode either, so we rely on it getting set here) and the settings provider will not register for this setting with DFCI, so we won't be able to change it, right?

Reply:
Override looked the dfci flow. I had an impression that this is above this blob. Will update.